api-inference-community/docker_images/spacy/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM tiangolo/uvicorn-gunicorn:python3.8
+LABEL maintainer="Omar Sanseviero omar@huggingface.com"
+
+# Add any system dependency here
+# RUN apt-get update -y && apt-get install libXXX -y
+
+COPY ./requirements.txt /app
+RUN pip install --no-cache-dir -r requirements.txt
+COPY ./prestart.sh /app/
+
+
+# Most DL models are quite large in terms of memory, using workers is a HUGE
+# slowdown because of the fork and GIL with python.
+# Using multiple pods seems like a better default strategy.
+# Feel free to override if it does not make sense for your library.
+ARG max_workers=1
+ENV MAX_WORKERS=$max_workers
+ENV HUGGINGFACE_HUB_CACHE=/data
```

Comment:
Are we sure there is not other cache ? We should probably add the `pip` location, no ?

Reply:
Good idea. This is normally set with `--cache-dir` instead of an environment variable, but let's use that :)

Reply:
Either way, we just need to be sure that we avoid hitting arbitrary disk location for anything that could be cached (otherwise pod loading will take longer time, on any pod start.)

Reply:
Done, raising errors now.